scripts/lint-openapi-spec.mjs

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+import { lint, loadConfig } from '@redocly/openapi-core';
+import assert from 'node:assert';
+
+const pathToApi = './docs/attempts-api/openapi.yml';
+const config = await loadConfig({ configPath: '' });
+const lintResults = await lint({ ref: pathToApi, config });
+
+assert(
+  !lintResults.length,
+  `The OpenAPI spec is not valid.
+
+  Found ${JSON.stringify(lintResults)}
```

Comment:
Non-blocking: The [third argument of `JSON.stringify`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#space) might be useful to pass to improve the output formatting:

```suggestion
  Found ${JSON.stringify(lintResults, null, 2)}
```